content/_data/authors/dheerajodha.adoc

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
would be good to bit a mini bio in here:

![image](https://user-images.githubusercontent.com/21194782/118389790-7cde8300-b623-11eb-9842-51de8d454537.png)

Reply:
Done, thanks